app/Http/Middleware/AuthMain.php

```diff
@@ -4,24 +4,33 @@
 
 use Closure;
 use Illuminate\Http\Response;
-use Cookie;
-use App\Models\User;
 use Illuminate\Support\Facades\Auth;
-use Emarref\Jwt\Claim;
 
 
 class AuthMain extends AuthService
 {
+    protected $urlAuth = 'http://team.binary-studio.com/auth/';
```

Comment:
Please move the urlAuth to a separated config file. Let's say: /config/authserver.php